larg4/pluginActions/ParticleListAction_service.cxx

```diff
@@ -77,7 +77,8 @@ namespace larg4 {
       fKeepEMShowerDaughters( p.get<bool>("keepEMShowerDaughters",true) ),
       fNotStoredPhysics( p.get< std::vector<std::string> >("NotStoredPhysics",{})),
       fkeepOnlyPrimaryFullTraj( p.get<bool>("keepOnlyPrimaryFullTrajectories",false) ),
-      fSparsifyTrajectories( p.get<bool>("SparsifyTrajectories",false) )
+      fSparsifyTrajectories( p.get<bool>("SparsifyTrajectories",false) ),
+      fSparsifyMargin( p.get<double>("SparsifyMargin",0.1) )
```

Comment:
Is a default parameter desired here?  
Not having one would make it easier to catch when the fcl param `SparsifyMargin` has been misconfigured.

Reply:
Yes -- valid suggestion. I have removed the default.